src/gui/qgsexpressionbuilderwidget.cpp

```diff
@@ -440,6 +443,26 @@ void QgsExpressionBuilderWidget::loadRecent( const QString &collection )
   }
 }
 
+void QgsExpressionBuilderWidget::loadLayers()
+{
+  QMap<QString, QgsMapLayer *> layers = QgsProject::instance()->mapLayers();
```

Comment:
If you replace the `QgsProject::instance()` call with a setter you help to make the future brighter.